src/index.ts

```diff
@@ -1,4 +1,6 @@
-import { codes } from './lib/codes';
+import { codes, retryableCodes } from './lib/codes';
 
 export * from './lib/codes';
 export const isValidCode = (code: string): boolean => code in codes;
+export const isRetryableCode = (code: string): boolean =>
+    code in retryableCodes;
```

Comment:
```suggestion
    retryableCodes.includes(code);
```

Reply:
well, `includes` not available in `es2015` so `indexOf() !== -1`?

Reply:
derp fixing